<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>632</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>632</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>475</width>
    <height>632</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="selectedAdresses_TableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="adressesValues_TableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="newSearch">
       <property name="text">
        <string>New Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextValue">
       <property name="text">
        <string>Next Value</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="valueLabel">
       <property name="text">
        <string>Value:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="valueLineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="typeComboBox">
       <item>
        <property name="text">
         <string>1 Byte</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 Bytes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 Bytes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8 Bytes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Float</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Double</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>475</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <addaction name="actionOpen_Process"/>
   </widget>
   <addaction name="menuProcess"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Process">
   <property name="text">
    <string>Open Process</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
